### DISCUSSION TITLE
xUnit test throws TypeInitializationException

### DISCUSSION THREAD

**Discussion by rocky050371** [xunit/xunit#2950] [Question] [answered]
I have posted the following on Stack Overflow, any help would be appreciated

https://stackoverflow.com/questions/78619545/xunit-test-throws-typeinitializationexception

**Comment by rocky050371** [comment, marked as answer]
The issue was resolved by clearing down DeploymentManagerAutoInitializer.cs, I am not sure where the file has appeared from (says it is Auto Generated) and can't find a great deal of info on it, it also cannot be removed from the project (along with WindowsAppSdk-VersionInfo.cs).

**Reply by bradwilson** [reply]
I'm glad you figured out the problem, as I have no experience with Maui (and that's definitely not a file we would have auto-generated).
